src/Helpers/HttpHelper.php

```diff
@@ -81,7 +81,7 @@ public function __construct(array $config = [])
     public function __call($method, $args)
     {
         $url = parse_url($args[0]);
-        $urlHost = $url['host'] ?? '';
+        $urlHost = $url['host'] ?? $this->clientConfig['base_uri'] ?? '';
```

Comment:
Veja se consegue usar o optional($this->clientConfig)['base_uri'] para evitar o duplo ternário

Reply:
mesmo se o cliente config for setado n quer dizer que vai ter o base_uri pode ter outros campos, nesse caso precisaria de mais um (talvez...)